torch/_functorch/_aot_autograd/subclass_utils.py

```diff
@@ -36,7 +36,7 @@ def requires_subclass_dispatch(args, fw_metadata: ViewAndMutationMeta) -> bool:
     return any_subclass_args or any_subclass_outputs
 
 
-def create_subclass_metadata(a, start_idx):
+def create_subclass_metadata(a: object, start_idx: int, count_symints: bool):
```

Comment:
nit: if we can't easily type `a`, we usually put `a: Any`

Reply:
That was probably my IDE. I'll change it.